lib/launcher.js

```diff
@@ -350,6 +350,7 @@ class Launcher {
                 const res = await got.head(this.settings.baseURL, opts)
                 statusCode = res.statusCode || 500
             } catch (error) {
+                console.log(`Failed to poll NR ${error}`)
```

Comment:
Would it not be beneficial to log out the error object so users can see the extra info like the status code and extended error info?

```suggestion
                console.log(`Failed to poll NR, error)
```